### ISSUE TITLE
Missing items, broken links in Markdown converted home page

### ISSUE THREAD

**Issue by nuxy** [ankitskvmdam/clean-jsdoc-theme#284]
**Describe the bug**

Using the theme provided JSDoc [configuration](https://ankdev.me/clean-jsdoc-theme/v4/#example-jsdoc-config) the resulting output contains the following errors:

1. [List item](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li) elements are missing the bullet points/ascending counter.

![Missing list-item](https://github.com/ankitskvmdam/clean-jsdoc-theme/assets/1234102/1376e9b2-8e03-4c58-a221-95574921261f)

2. In the Table of Content links (**On this page**) the anchor is missing the ID target. Clicking on the link results with console message `Empty string passed to getElementById()`. Note: This _only occurs in the home page_.

3. Theme options don't seem to be honored. For example, the following doesn't update the HTML `<title>Home</title>`

```
"theme_opts": {
  "homepageTitle": "Clean JSDoc theme"
}
```

**To Reproduce**

The issues can be [seen here](https://lambda-lambda-lambda.github.io/doc)

**Expected behavior**

The issues listed above do not occur.

**Screenshots**

**Desktop:**

 - Ubuntu `22.04`
 - Firefox (`125.0.1`), Brave (`1.65.114`)

**Additional context**

NPM: `10.2.3`
Node: `v20.11.0`
JSDoc: `4.0.2`

**Comment by ankitskvmdam** [comment]
Hi @nuxy ,

Thanks for creating this issue. I am working on this. Will update you soon.

**Comment by ankitskvmdam** [comment]
@nuxy
1. No bullet for list items is fixed in `v4.3.0`
2. To fix this you need to add the following in you jsdoc config file. For more visit [here](https://github.com/ankitskvmdam/clean-jsdoc-theme?tab=readme-ov-file#example-jsdoc-config).

```json5
"markdown": {
        "hardwrap": false,
        "idInHeadings": true // <- otherwise no id will be attached to the headings.
    }
```
3.  `homepageTitle` will only update the title of the browser's tab when user is viewing the homepage. If you want to update the title of the sidebar then you need to set `title`.

![image](https://github.com/ankitskvmdam/clean-jsdoc-theme/assets/22586210/8e5bf2ac-832f-4ea3-ad41-e2c8873d97e0)

**Comment by nuxy** [comment]
Hi @ankitskvmdam -

Thanks for addressing this so quickly. I have tested the [latest release](https://www.npmjs.com/package/clean-jsdoc-theme) everything works as expected.

Closing this issue.
